features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/SyslogReceiverNioThreadPoolImpl.java

```diff
@@ -189,6 +224,17 @@ public void run() {
         // Get a log instance
         Logging.putPrefix(Syslogd.LOG4J_CATEGORY);
 
+        ConsoleReporter reporter = ConsoleReporter.forRegistry(METRICS)
+            .convertRatesTo(TimeUnit.SECONDS)
+            .convertDurationsTo(TimeUnit.MILLISECONDS)
+            .build();
+        reporter.start(1, TimeUnit.SECONDS);
```

Comment:
Can you delete these ConsoleReporter calls? We don't want them at runtime since they print to stdout. They were only useful for testing.